Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -171,10 +167,6 @@ WarpXParticleContainer::AddNParticles (int /*lev*/,
         particle_tile.push_back_real(PIdx::uy,     vy + ibegin,     vy + iend);
         particle_tile.push_back_real(PIdx::uz,     vz + ibegin,     vz + iend);
 
-        if ( (NumRuntimeRealComps()>0) || (NumRuntimeIntComps()>0) ){
-            auto& ptile = DefineAndReturnParticleTile(0, 0, 0);
-        }
-
```

Comment:
Fixes the warning:
`./Source/Particles/WarpXParticleContainer.cpp:175:19`: warning: unused variable ‘ptile’ [-Wunused-variable]
             auto& ptile = DefineAndReturnParticleTile(0, 0, 0);

Reply:
same as above, just discard the return value instead.